src/ui/public/timepicker/timepicker.less

```diff
@@ -0,0 +1,19 @@
+.kbn-timepicker {
+  [kbn-time-input] {
+    text-align: center;
+  }
+  .kbn-timepicker-modes {
+    text-transform: capitalize;
+  }
+  .kbn-timepicker-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-refresh-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-timepicket-alert {
+    width: 100px;
+  }
+}
```

Comment:
@cjcenizal Hmm I think that it is good to limit the scope of this classes to be applied only inside kbn-timepicker directive. 
If these classes should apply anywhere else then probably they should not be in timepicker.less file but in some other file like kibana.less
Maybe we could rename these classes and drop the **kbn-timepicker** or just **kbn-** prefix instead ?